app/views/groups/new.html.erb

```diff
@@ -15,7 +15,7 @@
             <a href="/">
                 <i class="fa fa-arrow-left fa-2x text-white" aria-hidden="true"></i>
             </a>
-            <h2 class="m-0 p-0">Create Category</h2>
+            <h2 class="m-0 p-0">Create New Category</h2>
```

Comment:
* Kindly stick to the design given as much as possible as it currently your :

> * Colors
> * Typography: font face, size and weight.
> * Layout: composition and space between elements. dont match the given design 
![image](https://github.com/shaaibu7/Budget-app/assets/52497246/44738c67-5fc7-4065-a8c2-0be789e31157)
![image](https://github.com/shaaibu7/Budget-app/assets/52497246/e71bdb30-5527-48a0-a1dd-5f512e4ba1f8)
![image](https://github.com/shaaibu7/Budget-app/assets/52497246/c5b9eab2-5a7a-4325-b5fe-727308e5a474)
* Also ensure you handle n +1 issues

Reply:
Great work. Good work solving the n+1 issues also